pyempaq/main.py

```diff
@@ -144,8 +151,17 @@ def main():
     """Manage CLI interaction and call pack."""
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "source", type=pathlib.Path,
-        help="The source file (pyempaq.yaml) or the directory where to find it.")
+        "source",
+        type=pathlib.Path,
+        help="The source file (pyempaq.yaml) or the directory where to find it.",
+    )
+    parser.add_argument(
+        "--version",
+        "-V",
+        action="version",
+        version=f"%(prog)s {pkg_resources.get_distribution('pyempaq').version}",
+        help="The current version of pyempaq.",
+    )
```

Comment:
added **--version** and **-V** option that will print the version of the module and exit